CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ The project was created with `vite.js` and is written in `vue3`.
 Our backend lives in the `server`-directory and mostly communicates via websockets.
 Both frontend and backend share the same `package.json`.
 
-For production, the frontend is build into `dist`-directory and the server (`express.js`) exposes the `dist` directory as the root of the endpoint.
+For production, the frontend is built into `dist`-directory and the server (`express.js`) exposes the `dist` directory as the root of the endpoint.
```

Comment:
So in this context, we are describing that the frontend has been compiled and placed in the `dist` directory, which indicated that the action is completed, which is why we should be using the word "built" here instead of "build"

If we want to use "build" you'd need to structure the sentence something like this:
"For production, we build the frontend into `dist`-directory..."

Reply:
Coming around to your way. Let's add "the"

```suggestion
For production, the frontend is built into the `dist`-directory and the server (`express.js`) exposes the `dist` directory as the root of the endpoint.
```